lacommunaute/forum_conversation/tests/tests_views.py

```diff
@@ -707,6 +708,23 @@ def test_queryset_filtered_on_tag(client, db, tag):
     assertNotContains(response, other_topic.subject)
 
 
+def test_queryset_for_tagged_topic(client, db, snapshot):
+    tags = ["buckley", "jeff"]
+    tagged_topic = TopicFactory(with_post=True, with_tags=tags)
+    untagged_topic = TopicFactory(with_post=True)
+
+    response = client.get(reverse("forum_conversation_extension:topics") + f"?tags={tags[0]}")
```

Comment:
Comme tu préfères, pour info cette syntaxe existe :
```suggestion
    response = client.get(reverse("forum_conversation_extension:topics"), {"tags": tags[0]})
```